Digitization/ARICHDigitizer_module.cc

```diff
@@ -79,14 +79,21 @@ namespace emph{
   	     std::unordered_map<int,int> fSensorMap;
 	     float fCalibration;    //to be fixed later 
 	     bool fIncludeNoise;	 
-
+	     bool fFillTree;
+	
+	     TTree *fTest;
+             std::vector<int> PDG,TID,BLOCK, GRANPAPDG;
```

Comment:
my preference would be for these to not be capitalised

Reply:
Will do, I'm also making a new pull request and I'll add it to it